aws/resource_aws_emr_instance_fleet.go

```diff
@@ -0,0 +1,193 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAwsEMRInstanceFleet() *schema.Resource {
+	return &schema.Resource{
```

Comment:
This new resource should implement import support, which can be implemented similar to the `aws_emr_instance_group` resource:

https://github.com/terraform-providers/terraform-provider-aws/blob/8fc7ebf0d5bc893921207efbbc4ea5696b147cde/aws/resource_aws_emr_instance_group.go#L29-L41

Additional details including acceptance testing and documentation can be found in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/contribution-checklists.md#adding-resource-import-support).